pom.xml

```diff
@@ -80,7 +86,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.8</minimum>
+                                            <minimum>0.7</minimum>
```

Comment:
<img width="917" alt="image" src="https://user-images.githubusercontent.com/68428976/164625749-77282a7d-0637-4035-950c-a295b0dc429e.png">
It's because you didn't complete the assignment.

Reply:
this task kills me)))))))